server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/qualitygate/QualityGateHolderImpl.java

```diff
@@ -19,44 +19,46 @@
  */
 package org.sonar.server.computation.task.projectanalysis.qualitygate;
 
-import com.google.common.base.Optional;
-import javax.annotation.CheckForNull;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.sonar.server.qualitygate.EvaluatedQualityGate;
 
-import static com.google.common.base.Optional.absent;
-import static com.google.common.base.Optional.of;
 import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
 public class QualityGateHolderImpl implements MutableQualityGateHolder {
-  private boolean initialized = false;
-  @CheckForNull
+  @Nullable
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "Optional" variables should not be "@Nullable". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2789)